### PR TITLE
Temporary fix for CI by pinning archspec version

### PR DESCRIPTION
**Pull request by TopRichard**
Following EESSI PR 482

### CODE REVIEW

File: easystacks/pilot.nessi.no/2023.06/eessi-2023.06-eb-4.9.0-2023a.yml [hunk 1]
```diff
@@ -45,3 +45,4 @@ easyconfigs:
   - SAMtools-1.18-GCC-12.3.0.eb
   - VCFtools-0.1.16-GCC-12.3.0.eb
   - BEDTools-2.31.0-GCC-12.3.0.eb
+  - ASE-3.22.1-gfbf-2023a.eb
```
Comment (trz42) [inline easystacks/pilot.nessi.no/2023.06/eessi-2023.06-eb-4.9.0-2023a.yml]: Looks like a leftover of another PR?

**Review by trz42 — CHANGES_REQUESTED** [review]
The changes to the CI scripts look fine. However, also an easyconfig is added to an easystack file. Was this intended?

File: easystacks/pilot.nessi.no/2023.06/eessi-2023.06-eb-4.9.0-2023a.yml (hunk 1 above)
Comment (TopRichard) [reply easystacks/pilot.nessi.no/2023.06/eessi-2023.06-eb-4.9.0-2023a.yml]: 100% it is a left over from another PR

**Review by trz42 — APPROVED** [review]
lgtm

**Pull request merged**